DigitalLearningSolutions.Web/Views/Shared/PaginatedPage/_BaseConfigurablePaginationNav.cshtml

```diff
@@ -0,0 +1,32 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.SearchablePage
+@model (IBasePaginatedViewModel pageModel, bool useAlertOnPageNumber)
+
+  <div class="nhsuk-pagination__list">
+    <div class="pagination-button-container">
+      <div class="nhsuk-pagination-item--previous" @(Model.pageModel.Page == 1 ? "hidden" : "")>
+        <button class="nhsuk-pagination__link nhsuk-pagination__link--prev pagination-button" type="submit" asp-action="@ViewContext.RouteData.Values["action"].ToString()" asp-route-page="@(Model.pageModel.Page - 1)">
+          <svg class="nhsuk-icon nhsuk-icon__arrow-left" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true">
+            <path d="M4.1 12.3l2.7 3c.2.2.5.2.7 0 .1-.1.1-.2.1-.3v-2h11c.6 0 1-.4 1-1s-.4-1-1-1h-11V9c0-.2-.1-.4-.3-.5h-.2c-.1 0-.3.1-.4.2l-2.7 3c0 .2 0 .4.1.6z"></path>
+          </svg>
+          <span class="nhsuk-pagination__title">Previous</span>
+        </button>
+      </div>
+    </div>
+    <div class="page-indicator-container">
+      <span @(Model.useAlertOnPageNumber ? "role=alert" : null)>
+        <span class="nhsuk-u-visually-hidden">page </span>
+        <span class="page-indicator" @(Model.pageModel.TotalPages <= 1 ? "hidden" : "")>@Model.pageModel.Page of @Model.pageModel.TotalPages</span>

```

Comment:
I've added the logic to hide this when there is 1 or 0 pages to fix the bug from 881.